css/properties/mask-composite.json

```diff
@@ -10,13 +10,14 @@
               "notes": "See also <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-composite'><code>-webkit-mask-composite</code></a> for a similar non-standard property that uses different keywords."
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": false,
+              "notes": "See also <a href='https://developer.mozilla.org/docs/Web/CSS/-webkit-mask-composite'><code>-webkit-mask-composite</code></a> for a similar non-standard property that uses different keywords."
```

Comment:
Why is this helpful?

Reply:
It looks like that was a note imported from MDN's old tables that was just left there.  I'm not too certain what purpose it serves, maybe someone else can explain?

Reply:
Then I strongly favor removing this.